src/Appwrite/Utopia/Response/Filters/V12.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Filters;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Filter;
+use Exception;
+
+class V12 extends Filter
+{
+    // Convert 0.13 Data format to 0.12 format
+    public function parse(array $content, string $model): array
+    {
+        $parsedResponse = $content;
+
+        switch ($model) {
+            // Update permissions
+            case Response::MODEL_ERROR:
+                $parsedResponse = $this->parseError($content);
+                break;
+            case Response::MODEL_SESSION:
+                $parsedResponse = $this->parseSession($content);
+
+            case Response::MODEL_FILE:
+                $parsedResponse = $this->parseFile($content);
+
+            case Response::MODEL_FUNCTION:
+                $parsedResponse = $this->parseFunction($content);
+
+            case Response::MODEL_EXECUTION:
+                $parsedResponse = $this->parseExecution($content);
+
+            case Response::MODEL_USAGE_BUCKETS:
+                $parsedResponse = $this->parseUsageBuckets($content);
+
+            case Response::MODEL_USAGE_STORAGE:
+                $parsedResponse = $this->parseUsageStorage($content);
+
```

Comment:
Are we sure these are the only models that changed? I think a lot more has changed around the storage api, the ErrorDev response model, deployments . Please double check them